### ISSUE TITLE
[tooltip] Additional content is not hoverable.

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#176]
According to WCAG 2.1 1.4.13, the content of the tooltip should be hoverable. This is not the case. Moving the mouse from the invoker to the tooltip makes the tooltip disappear.

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html
